recipe/meta.yaml

```diff
@@ -1,35 +1,35 @@
 {% set name = "bandit" %}
-{% set version = "1.7.8" %}
+{% set version = "1.7.9" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 36de50f720856ab24a24dbaa5fee2c66050ed97c1477e0a1159deab1775eab6b
+  sha256: 7c395a436743018f7be0a4cbb0a4ea9b902b6d87264ddecf8cfdc73b4f78ff61
 
 build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
   entry_points:
     - bandit=bandit.cli.main:main
     - bandit-config-generator=bandit.cli.config_generator:main
     - bandit-baseline=bandit.cli.baseline:main
-  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
+    - python >=3.8
     - pbr >=2.0.0
     - pip
-    - python
     - setuptools
   run:
-    - colorama >=0.3.9  # [win]
-    - GitPython >=1.0.1
```

Comment:
the baseline extra needs GitPython>=3.1.30, see https://github.com/PyCQA/bandit/blob/main/setup.cfg#L39

Please keep python first here and the rest in alphabetical order